test/configuration-plugins.js

```diff
@@ -11,7 +11,9 @@ var join = path.join
 var fixtures = join(__dirname, 'fixtures')
 
 test('configuration', function (t) {
-  t.plan(6)
+  var node = Number.parseInt(process.versions.node.split('.')[0], 10)
+
+  t.plan(node < 12 ? 6 : 9)
```

Comment:
manual parsing feels a bit off here.
would it make sense to do something along the lines of:

```suggestion
  const node = process.versions.node

  t.plan(semver.lt(node, '12.0.0') ? 6 : 9)
```

Reply:
good idea, added!